libs/blocks/section-metadata/section-metadata.js

```diff
@@ -1,35 +1,34 @@
-function handleBackground(div, section) {
-  const pic = div.background.content.querySelector('picture');
-  if (pic) {
-    section.classList.add('has-background');
-    pic.classList.add('section-background');
-    handleFocalpoint(pic, div.background.content);
-    section.insertAdjacentElement('afterbegin', pic);
-  } else {
-    const color = div.background.content.textContent;
-    if (color) {
-      section.style.background = color;
-    }
-  }
-}
-
 export function handleFocalpoint(pic, child, removeChild) {
   const image = pic.querySelector('img');
   if (!child || !image) return;
   let text = '';
   if (child.childElementCount === 2) {
     const dataElement = child.querySelectorAll('p')[1];
     text = dataElement?.textContent;
-    removeChild ? dataElement?.remove() : '';
+    if (removeChild) dataElement?.remove();
   } else if (child.textContent) {
     text = child.textContent;
     const childData = child.childNodes;
-    removeChild ? childData.forEach((c) => c.nodeType === Node.TEXT_NODE && c.remove()) : '';
+    if (removeChild) childData.forEach((c) => c.nodeType === Node.TEXT_NODE && c.remove());
   }
   const directions = text.trim().toLowerCase().split(',');
   const [x, y = ''] = directions;
   image.style.objectPosition = `${x} ${y}`;
 }
+function handleBackground(div, section) {
+  const pic = div.background.content.querySelector('picture');
```

Comment:
I know this was just copied over, but it stands out as a potential risk - are there any safety mechanisms to check that `div.background.content` actually exists? Could we use some optional chaining here for safety?

Reply:
agreed, good call